libs/host/Garnet.host.csproj

```diff
@@ -29,4 +31,12 @@
     <InternalsVisibleTo Include="Garnet.test" Key="0024000004800000940000000602000000240000525341310004000001000100011b1661238d3d3c76232193c8aa2de8c05b8930d6dfe8cd88797a8f5624fdf14a1643141f31da05c0f67961b0e3a64c7120001d2f8579f01ac788b0ff545790d44854abe02f42bfe36a056166a75c6a694db8c5b6609cff8a2dbb429855a1d9f79d4d8ec3e145c74bfdd903274b7344beea93eab86b422652f8dd8eecf530d2" />
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <!-- Include files from 8.0 root directory and runtimes in Nuget pacakge - don't want other folders in 8.0 root though  -->
+    <Content Include="..\host\bin\Release\net8.0\*" Pack="true" PackagePath="lib\net8.0" />
```

Comment:
do we need to include _everything_ in this path?

Reply:
This was from Marc so I am not 100% sure but I believe so.

Reply:
We get .pdb in library AND garnetserver nuget packages (as well as pdbs in the symbols package). This is what I was talking about offline. In earlier version of library we did NOT get the .pbs in Nuget package directly but we are now.

Reply:
We decided to go with embedding the debug symbols in the DLLs, so there will not be any .snupkg files